python/ray/_private/runtime_env/working_dir.py

```diff
@@ -157,6 +198,10 @@ async def create(
         local_dir = await download_and_unpack_package(
             uri, self._resources_dir, self._gcs_aio_client, logger=logger
         )
+        if is_whl_uri(uri):
+            local_dir = await self._unpack_wheel(
```

Comment:
It's a bit confusing that we "download and unpack" and then "unpack" again.  Are the names accurate? Feel free to rename them

Reply:
Or do we not need to download_and_unpack if it's a whl uri?

Reply:
ohh okk, I can rename it. The `download_and_unpack_package` is just unpacking the zip file so need to extract the .whl file afterward (which were done by `download_and_unpack_wheel` function before that I refactored